content/en-US/api/CanvasRenderingContext2D.reset.md

```diff
@@ -13,7 +13,7 @@ browser_compatibility: api.CanvasRenderingContext2D.reset
 ## Description
 
 The `reset()` method of the `CanvasRenderingContext2D` interface
-resets the rendering context to its default state. This includes setting the backing buffer to transparent black, clearing the saved state stack, clearing the default path and resetting the drawing state to its initial values.
+resets the rendering context to its default state. This includes setting the [output bitmap](https://html.spec.whatwg.org/multipage/canvas.html#output-bitmap) to transparent black, clearing the saved [state stack](https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-state:canvasstate), clearing the [current default path](https://html.spec.whatwg.org/multipage/canvas.html#current-default-path) and resetting the drawing state to its initial values.
```

Comment:
The audience for this is web developers, not browser implementors. Are there links from [MDN](https://developer.mozilla.org/en-US/) that you can use?

Reply:
Unfortunately these things are much less rigorously defined on MDN. Would this be acceptable definition for "state stack"?:
  https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D#the_canvas_state

As for the output bitmap, it's referenced, but never defined. Like [here](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D#compositing):

> With `globalAlpha` applied this sets how shapes and images are drawn onto the existing bitmap.

Or [here](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D#pixel_manipulation):

> Paints data from the given ImageData object onto the bitmap. If a dirty rectangle is provided, only the pixels from that rectangle are painted.

[Here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas#accessibility_concerns) is probably the closest we get to the idea that the "canvas _is_ a bitmap", but it's just a note about accessibility concerns:

> The <canvas> element on its own is just a bitmap and does not provide information about any drawn objects.

Reply:
Are there definitions anywhere other than the spec? If not, the solution is to fix MDN. For web developers the spec should be the fallback, not the primary means of conveying information. As I said different audiences.

Reply:
Nope. What I commented above are the only references I can find for the output bitmap. 

I now realize that I was writing in a bit more of a "spec-ese" style and things like "current default path" can just be "stored path operations." For that there's a clear MDN link.

I think I was also perhaps being too formal for "output bitmap" when in developer land this is just kind of the "canvas" itself. I changed the language so that perhaps we don't need an explicit reference.

Reply:
All of that sounds good. If there's anything else, leave the spec defs for now. Can you help me identify what needs to be improved to link to MDN?